mission_manager/src/behaviors/waypoint.cpp

```diff
@@ -175,157 +169,101 @@ void WaypointBehavior::latLongtoUTM(double latitude, double longitude, double* p
 #endif
```

Comment:
@patapufate hmm this block seems to be compiled out?

Reply:
Yes. Good Catch.